File: Ohtap E Commerce Domain Models/Development/PHP/Objects/RegionDetails/RegionDetail.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Regions;

use Ohtap\ECommerce\DomainModels\Objects\Languages\ILanguage;

class RegionDetail  implements IRegionDetail
{


    public  ILanguage $Language;
    public string $Description;
    public int $State;
    public int $OrderNo;
    public string $ID;
    public string $Name;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/RegionDetails/RegionDetail.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Languages;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Languages;
using Ohtap.ECommerce.Domain.Models.Objects.RegionDetails;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.RegionDetails
{
    public class RegionDetail : RegionDetailBasic, IRegionDetail
    {
        ILanguage IRegionDetail.Language { get; set; }
    }
}
